src/Pages/Profile.jsx

```diff
@@ -278,6 +279,8 @@ export default function GitHubProfile() {
                     <StatCard value={userData.following} label="Following" icon={Users} />
                 </div>
 
+                <ProjectContributions projectContributions={userData.projectContribution} />
```

Comment:
it should be `projectContributions`, also destructure `projectContribution` from get request data and set it in userData from useEffect.